game/script-topics.rpy

```diff
@@ -1044,61 +1044,238 @@ init 5 python:
+    extend 3fcsgs " I mean it!"
+    n 4cllfl "Seriously -{w=0.5}{nw}"
+    extend 4tsqem " unless you feel like making friends with your nearest chiropractor,{w=0.5}{nw}"
+    extend 4cslem " trust me when I say you {i}really{/i} don't wanna spend all your time slouching."
+    n 2csqss "...Or hunched up on your chair like some kind of weird computer gremlin."
+    n 6unmaj "If you're doing it right,{w=0.5}{nw}"
+    extend 3ullaj " then you should have your arms and thighs parallel to the ground,{w=0.5}{nw}"
+    extend 3fcssm " with your eyes roughly at the top of your screen.{w=0.75}{nw}"
+    extend 3fcsbg " Easy peasy!"
+    n 4unmfl "Oh,{w=0.2} right -{w=0.5}{nw}"
+    extend 2clrsssbl " make sure your feet can still touch the floor,{w=0.2} though.\n{w=0.75}{nw}"
+    extend 2fcscasbr "Even {i}I{/i} can do that,{w=0.2} [player]."
+    n 1fnmaj "Number two:{w=0.75}{nw}"
+    extend 2fcsbg " distance!"
+    n 2clrbg "It's pretty easy to forget,{w=0.5}{nw}"
+    extend 4tnmbo " but if you wanna avoid sore eyes then you've gotta make sure you're actually a decent distance from the screen too."
+    n 2fcsca "Not right up close or so far away you need binoculars!"
 
     if Natsuki.isEnamored(higher=True):
-        n 3kwmsml "But you know I only do these things because I really care about you,{w=0.1} [player]...{w=0.3} right?"
-        n 4kwmnvl "So please...{w=0.3} take care of yourself, okay?{w=0.2} I don't want you hurting because of me."
+        n 2fcsbglsbr "I-{w=0.2}I know you just can't get enough of me,{w=0.2} [player].{w=0.75}{nw}"
+        extend 4cslfllsbr " But really."
+        n 4ccspol "Even I don't wanna see you practically pressing your face against the screen either."
 
+    elif Natsuki.isAffectionate(higher=True):
+        n 2fsrsslsbr "I-{w=0.2}I know I {i}am{/i} pretty awesome,{w=0.5}{nw}"
+        extend 4csqflsbr " but the last thing I wanna be greeted by is your face right up against the screen either."
+
+    else:
+        n 2ccsflsbr "A-{w=0.2}and besides,{w=0.5}{nw}"
+        extend 2cslflsbr " the last thing I wanna see is your face all smushed up against the screen,{w=0.2} [player].{w=0.75}{nw}"
+        extend 2ccsposbr " I didn't ask for {i}that{/i}."
+
+    n 2ulraj "So...{w=1}{nw}"
+    extend 4tsgbg " just make sure you're sitting about an arm's length away from the screen.{w=0.75}{nw}"
+    extend 4ccsbg " That's all I'm saying!"
+    n 3unmpu "Don't forget to keep all your stuff in easy reach though -{w=0.5}{nw}"
+    extend 3ccsbg " unless you're a psychic,{w=0.5}{nw}"
+    extend 6csqsm " you aren't getting far if you can barely hit the keyboard."
+    n 7ccsbg "Now,{w=0.2} number three:{w=0.75}{nw}"
+    extend 3fchbg " breaks!"
+    n 3tllfl "I don't know about you [player],{w=0.5}{nw}"
+    extend 7tnmbo " but personally?{w=0.75}{nw}"
+    extend 7csrem " I can't stand being stuck in one place for hours at a time."
```

Comment:
ngl that's a bit ironic given her situation. I could imagine that would encourage players even more to demand new places to visit.

Reply:
True; I can reword it to emphasise her stance on sitting specifically (as she at least gets up from her desk when the player isn't around, or goes to the closet to fetch things)